packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -251,19 +323,63 @@ const Select = <
             return null;
           }
 
-          return <Chip key={item} label={selectedOption.text} />;
+          return (
+            <MuiChip
+              key={item}
+              label={
+                <>
+                  {selectedOption.text}
+                  {!isPlaceholder &&
+                    controlledStateRef.current === CONTROLLED &&
+                    hasMultipleChoices && (
+                      <PlaceholderIcon
+                        odysseyDesignTokens={odysseyDesignTokens}
+                      />
+                    )}
+                </>
+              }
+              tabIndex={-1}
+              onDelete={
+                isPlaceholder
+                  ? controlledStateRef.current === CONTROLLED
+                    ? () => removeSelection(item)
+                    : undefined
+                  : undefined
+              }
```

Comment:
When I see nested ternaries and two `undefined` values, I think that's a place we can improve this.

```suggestion
              onDelete={
                isPlaceholder && controlledStateRef.current === CONTROLLED
                  ? () => removeSelection(item)
                  : undefined
              }
```